vipps-ecom-api.md

```diff
@@ -1437,21 +1437,21 @@ is always to query the eCom API once you arrive back in the merchant app/website
 
 The sections below explain in more detail how to integrate for browsers and apps.
 
-## App-switch between browser and the Vipps app
+## App-switching
+1. The merchant need to pass the URL scheme of the app into the `fallBack` field in the [initiate payment request](#initiate-payment-flow-api-calls). See [App-switch on iOS](#app-switch-on-ios) and [App-switch on Android](#app-switch-on-android) for specifics.
+2. The merchant app should open the Vipps app deeplink received from the [initiate payment request](#initiate-payment-flow-api-calls).
+3. Once the operation in the Vipps app is completed, Vipps will redirect the user to the deeplink specified in the `fallBack` field.
+4. The merchant app should query the [payment details endpoint](#get-payment-details) for updated status on the payment once user returns from the Vipps app.
 
-For mobile and desktop browsers, integration is handled by Vipps using the Vipps landing page.
-
-The merchant needs to provide a valid `fallBack` URL.
-When Vipps has completed the operation, the `fallBack` URL will be opened in the browser.
-To maintain the session, the merchant can pass along a session identifier in the `fallBack` URL.
+**Please note:** Vipps will append a status at the end of the fallback URL. For example, if your `fallBack` URL is `merchantApp://result?myAppData`, Vipps
```

Comment:
Just remove this, best not to mention it at all

Reply:
Thought about it, still on the fence. I think merchants would find it confusing once they see the status in the actual deeplinks, and they might even append things at the end themselves, causing even more confusion when parsing.

Reply:
I can confirm what Markus says. We have gotten some (not a lot) merchants that see they get ?status= and want to use it. As long as we send it, merchants will notice, so I think it's better to clearly state that they SHOULD NOT USE IT.

Reply:
We too often have merchant who don't understand deprecation, if they ask just tell them its legacy and we won't reveal what it is because we don't want them to use it. They can try to backwards engineer it but 🤷‍♂️

Reply:
Some will not ask, just see status, understand that it's status and start using it.

Reply:
We want to avoid "if they ask". I think most of the merchants integrating app-to-app will notice this and be confused. I still think it's best to inform the customers that they will receive it, but that they should not use it. If we still get lots of questions or issues about it, we can see if removing it would help

Reply:
Ok, then the text should say "a deprecated legacy field 'status' is added as a query parameter the the fallback. THIS SHOULD NOT BE USED, IT MAY BE REMOVED AT ANY TIME WITHOUT WARNING"

I only want this here if it is crystal clear that we will remove it. I am very tempted to tell the apps to remove this field.

Reply:
How would we go about removing that status? We don't have any idea how many merchants may be depending on this. I'm fine with adding an even clearer text, but not so sure about adding the "MAY BE REMOVED AT ANY TIME WITHOUT WARNING" notice

Reply:
We would send out a communication in the Dev news letter, and a email to merchants using isApp: true, then remove 6 months later